setup.py

```diff
@@ -1,14 +1,11 @@
-import os
 from setuptools import setup, find_packages
+import subprocess
 
 
 def get_version():
-    basedir = os.path.dirname(__file__)
-    with open(os.path.join(basedir, 'rq_dashboard/version.py')) as f:
-        locals = {}
-        exec(f.read(), locals)
-        return locals['VERSION']
-    raise RuntimeError('No version info found.')
+    """Return the latest git tag from repository in cwd."""
```

Comment:
Oh ok. What's the use-case that makes this a problem?

I use https://pypi.python.org/pypi/hgtools for my software in mercurial. It takes the version from the Hg tag. So this is a poor man's equivalent for git instead.

We unavoidably have version numbers in Git and the package itself (discoverable with pkg_resources from setuptools). We don't really want a third location. I've used files containing the version number before in the past, and it is so easy to forget to keep it in sync with the VC tag.

I suppose we could create a fab task to do the tagging. It would need to set the version in a file, add and check-in that file, then create the git tag. Seems fussy to me.

How strongly do you feel about it?